pkg/ui/workspaces/cluster-ui/src/insights/workloadInsights/util/insightsColumns.tsx

```diff
@@ -44,115 +48,112 @@ export function getLabel(
-        {getLabel("query", execType)}
-      </Tooltip>
-    );
+    return makeToolTip(<p>tooltipText</p>, "query", execType);
   },
   insights: (execType: InsightExecEnum) => {
-    return (
-      <Tooltip
-        placement="bottom"
-        style="tableTitle"
-        content={
-          <p>
-            The category of insight identified for the {execType} execution.
-          </p>
-        }
-      >
-        {getLabel("insights")}
-      </Tooltip>
+    return makeToolTip(
+      <p>The category of insight identified for the {execType} execution.</p>,
+      "insights",
     );
   },
   startTime: (execType: InsightExecEnum) => {
-    return (
-      <Tooltip
-        style="tableTitle"
-        placement="bottom"
-        content={<p>The timestamp at which the {execType} started.</p>}
-      >
-        {getLabel("startTime")}
-      </Tooltip>
+    return makeToolTip(
+      <p>The timestamp at which the {execType} started.</p>,
+      "startTime",
     );
   },
   elapsedTime: (execType: InsightExecEnum) => {
-    return (
-      <Tooltip
-        style="tableTitle"
-        placement="bottom"
-        content={
-          <p>The time elapsed since the {execType} started execution.</p>
-        }
-      >
-        {getLabel("elapsedTime")}
-      </Tooltip>
+    return makeToolTip(
+      <p>The time elapsed since the {execType} started execution.</p>,
+      "elapsedTime",
     );
   },
   username: (execType: InsightExecEnum) => {
-    return (
-      <Tooltip
-        style="tableTitle"
-        placement="bottom"
-        content={<p>The user that opened the {execType}.</p>}
-      >
-        {getLabel("username")}
-      </Tooltip>
-    );
+    return makeToolTip(<p>The user that opened the {execType}.</p>, "username");
   },
   applicationName: (execType: InsightExecEnum) => {
-    return (
-      <Tooltip
-        style="tableTitle"
-        placement="bottom"
-        content={<p>The name of the application that ran the {execType}.</p>}
-      >
-        {getLabel("applicationName")}
-      </Tooltip>
+    return makeToolTip(
+      <p>The name of the application that ran the {execType}.</p>,
+      "applicationName",
     );
   },
   numRetries: () => {
-    return (
-      <Tooltip
-        style="tableTitle"
-        placement="bottom"
-        content={"The number of times this statement encountered a retry."}
-      >
-        {getLabel("numRetries")}
-      </Tooltip>
+    return makeToolTip(
+      <p>The number of times this statement encountered a retry.</p>,
+      "numRetries",
+    );
+  },
+  isFullScan: () => {
+    return makeToolTip(
+      <p>
+        The table is scanned on all key ranges of the rides_pkey index (i.e., a
+        full table scan).
+      </p>,
+      "isFullScan",
+    );
+  },
+  contention: () => {
+    return makeToolTip(
+      <p>
+        Lock contention happens when multiple processes are trying to access the
```

Comment:
Use a similar description we have on SQL Activity (without saying average, since it doesn't apply here or the part about the gray bar), with the link to contention docs
https://github.com/cockroachdb/cockroach/blob/08f810fbdf6b88a3a072189b50f855bd0df329c1/pkg/ui/workspaces/cluster-ui/src/statsTableUtil/statsTableUtil.tsx#L610-L616